sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -266,7 +266,7 @@
    * deduplication prevents from receiving the same exception multiple times when there is more than
    * one framework active that captures errors, for example Logback and Spring Boot.
    */
-  private Boolean enableDeduplication = true;
+  private @Nullable Boolean enableDeduplication = true;
```

Comment:
Since we have a default, can't we get away with this being `NotNull`?

Reply:
This is unfortunately coming from binding external configuration. I think it was a mistake to bind external config directly to `SentryOptions` - there should be an object dedicated to it, which can be applied to `SentryOptions`. 

I am happy to prepare a PR for it in case you believe it makes sense.

Reply:
if we change it to `boolean` is it possible to bind? assuming in that case it would either set as-is (false) or set the value if it existed.

We can discuss changing this but for the sake of this PR then I guess its fine as Nullable if no other way

Reply:
If we change it to `boolean` we will not know if it was set externally to true, or if its a default value. Leaving a nullable then.